research/src/pages/popup/popover.research.explainer.mdx

```diff
@@ -358,50 +356,47 @@ This not only allows developer ease-of-use from Javascript, but also allows for
 
 ```javascript
 function supportsPopover() {
-  return HTMLElement.prototype.hasOwnProperty("popover");
+  return HTMLElement.prototype.hasOwnProperty('popover')
 }
 ```
 
 Further, only [valid values](#html-content-attribute) of the content attribute will be reflected to the IDL property, with invalid values being reflected as the value `manual`. For example:
 
 ```javascript
-const div = document.createElement('div');
-div.setAttribute('popover','AUTO');
-div.popover === 'auto'; // true (note lowercase)
-div.setAttribute('popover','invalid!');
-div.popover === 'manual'; // true
-div.removeAttribute('popover');
-div.popover === null; // true
+const div = document.createElement('div')
+div.setAttribute('popover', 'AUTO')
+div.popover === 'auto' // true (note lowercase)
+div.setAttribute('popover', 'invalid!')
+div.popover === 'manual' // true
+div.removeAttribute('popover')
+div.popover === null // true
 ```
 
 This allows feature detection of the values, for forward compatibility:
 
 ```javascript
 function supportsPopoverType(type) {
-  if (!HTMLElement.prototype.hasOwnProperty("popover"))
-    return false; // Popover API not supported
-  const testElement = document.createElement('div');
-  testElement.popover = type;
-  return testElement.popover == type.toLowerCase();
+  if (!HTMLElement.prototype.hasOwnProperty('popover')) return false // Popover API not supported
```

Comment:
Disagree with this one too - having a control flow statement that far from the front of the line is a bug waiting to happen.